src/server/iframe.html.js

```diff
@@ -3,6 +3,11 @@ export default function (headHtml) {
     <!DOCTYPE html>
     <html>
       <head>
+        <script>
+          if (window.parent !== window) {
```

Comment:
this might actually throw an exception if the parent isn't same-origin - can you test it on a cross-origin page first? you may need to add a try/catch, or some other check.